vhdbuilder/packer/init-variables.sh

```diff
@@ -105,6 +105,14 @@ if [[ "$MODE" == "gen2Mode" ]]; then
 		if [[ "$OS_SKU" == "CBLMariner" ]]; then
 			SIG_IMAGE_NAME=${OS_SKU}${OS_VERSION//./}Gen2
 		fi
+
+		if [[ "$OS_TYPE" == "Windows" ]]; then
```

Comment:
In sigMode, SIG_IMAGE_NAME is the final created sig image name for test so I think that we should use a different name for gen2 as the sig image we create based on the raw vhd in sigMode

Reply:
Maybe we can use a GEN2_SIG_IMAGE_NAME and pass it to the pipeline steps in .builder-release-template-windows.yaml? But then lots of changes need to be made in this file at least. Will that clutter the code?

Reply:
Will reuse ``SIG_IMAGE_NAME`` and set the values differently for ``GEN2_SIG_FOR_PRODUCTION`` and ``GEN2_SIG_FOR_TEST``.